bootstrap_vagrant.sh

```diff
@@ -0,0 +1,67 @@
+#
+# Pretty CLI
+#
+echo "export CLICOLOR=1" >> /home/vagrant/.bashrc
+echo "export LSCOLORS=GxFxCxDxBxegedabagaced" >> /home/vagrant/.bashrc
+echo 'export PS1="[\u@\[\e[32;1m\]\H \[\e[0m\]\w]\$ "' >> /home/vagrant/.bashrc
+
+# Always start us in /vagrant
+echo "cd /vagrant/" >> /home/vagrant/.bashrc
+
+# Add some helpful aliases
+echo "alias runserver='cd /vagrant && python manage.py runserver [::]:8989'" >> /home/vagrant/.bashrc
+
+source /home/vagrant/.bashrc
+
+set -x
+apt-get update -y 
+# apt-get upgrade -y 
+apt-get install python-software-properties libxml2-dev libxslt1-dev -y
+apt-get install git vim python-dev libjpeg8 libjpeg8-dev rabbitmq-server curl vim screen -y
+apt-get install python-psycopg2 libpq-dev postgresql -y
+
+apt-get purge python-pip -y
+
+sudo -u postgres createuser --createdb --no-superuser --no-createrole modernomad
+sudo -u postgres createdb --owner=modernomad modernomadb
+sudo -u postgres psql -d postgres -c "ALTER USER modernomad WITH PASSWORD 'somepassword'" 
+sed -i.bak 's/peer$/trust/' /etc/postgresql/9.1/main/pg_hba.conf 
+
+if [ -f /etc/init.d/nginx ]; then
+    apt-get remove nginx
+    killall nginx
+fi
+
+cd /tmp
+curl -LO https://bootstrap.pypa.io/get-pip.py
```

Comment:
@jessykate Can you checkout the branch and verify that your local copy has this URL? This is correct and I don't see how you would get that error if this file is up to date.